rpcd/playbooks/roles/rpc_pre_upgrade/tasks/remove_beaver.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
This has already been mentioned, but these tasks need to target the containers and hosts beaver is installed on. Not just localhost.